geocity/apps/core/static/js/permit_request_amending.js

```diff
@@ -16,7 +25,8 @@ notify.addEventListener('change', (e) => {
 
 changeReasonDisplay()
 
-if (parseInt(statusChange.value) === COMPLEMENTARY_INFO) {
+if (parseInt(statusChange.value) === STATUS_AWAITING_SUPPLEMENT ||
+  parseInt(statusChange.value) === STATUS_RECEIVED) {
   statusChange.dispatchEvent(new Event("change"))
 }
 
```

Comment:
This does not work as expected: checkbox is always check when visible

![image](https://user-images.githubusercontent.com/3356536/188895352-6e11c9c8-3d98-4519-a942-4de571f1705c.png)

Reply:
It wasn't the case before my change?

Reply:
I review the logical behind it